commons-extension-webapp/src/main/webapp/WEB-INF/gatein-resources.xml

```diff
@@ -531,13 +531,23 @@
         (function() {
           <include>/javascript/purify.min.js</include>
           Vue.directive('sanitized-html',function(el, binding) {
-          el.innerHTML = DOMPurify.sanitize(Autolinker.link(binding.value, {newWindow: true}), {USE_PROFILES: {html: true},ADD_ATTR: ['target']});
+          el.innerHTML = DOMPurify.sanitize(Autolinker.link(binding.value, {newWindow: true}),
+                         {USE_PROFILES: {html: true}, ADD_TAGS: ["iframe"], ADD_ATTR: ['target','allow', 'allowfullscreen', 'frameborder', 'scrolling']});
             DOMPurify.addHook('afterSanitizeAttributes', function (node) {
             // add noopener attribute to external links to eliminate vulnerabilities
-            if ('target' in node) {
-             node.setAttribute('rel', 'noopener');
+             if ('target' in node) {
+               node.setAttribute('rel', 'noopener');
+             }
+            });
+            DOMPurify.addHook('uponSanitizeElement', function (node) {
+             if (node.tagName === 'iframe') {
+               const src = node.getAttribute('src') || '';
+               if (!src.startsWith('https://www.youtube.com/embed/')
+                           || !src.startsWith('https://player.vimeo.com/video/') || !src.startsWith('https://www.dailymotion.com/embed/video/')) {
```

Comment:
you will need to consider shorten URLS for Youtube, VIMEO and Dailymotion too

Reply:
I used only this **urls** because they are the only used by the embed video plugin even when past a shorten url in the `videodialog` they will be converted in the embedded code: https://github.com/exoplatform/news/blob/develop/webapp/src/main/webapp/js/ckeditor/plugins/video/dialogs/videoDialog.js#L76
But I still need your opinion should we keep this control or **remove it?** because only these types will be allowed no else.

Reply:
It is Ok, they are the only supported providers.
The shorten URL are already hadled here : https://github.com/exoplatform/news/blob/a3dbf84b3e3c1cbb17c117a0dfc7e3206f565a5a/webapp/src/main/webapp/js/ckeditor/plugins/video/dialogs/videoDialog.js#L104